From: Juergen Borleis <jbe@pengutronix.de>
Date: Mon, 23 Nov 2015 11:14:34 +0100
Subject: [PATCH] Skov-DO63x: use a more descriptive file name

Give the device tree files a name which is more descriptive for which board and
revision they are dedicated for.

Signed-off-by: Juergen Borleis <jbe@penugtronix.de>
---
 arch/arm/boot/dts/imx6dl-cpuA.dts             | 23 -------------------
 arch/arm/boot/dts/imx6dl-cpuB.dts             | 23 -------------------
 arch/arm/boot/dts/imx6dl-skov-dol63x-revA.dts | 23 +++++++++++++++++++
 arch/arm/boot/dts/imx6dl-skov-dol63x-revB.dts | 23 +++++++++++++++++++
 arch/arm/boot/dts/imx6q-cpuA.dts              | 32 ---------------------------
 arch/arm/boot/dts/imx6q-cpuB.dts              | 32 ---------------------------
 arch/arm/boot/dts/imx6q-skov-dol63x-revA.dts  | 32 +++++++++++++++++++++++++++
 arch/arm/boot/dts/imx6q-skov-dol63x-revB.dts  | 32 +++++++++++++++++++++++++++
 8 files changed, 110 insertions(+), 110 deletions(-)
 delete mode 100644 arch/arm/boot/dts/imx6dl-cpuA.dts
 delete mode 100644 arch/arm/boot/dts/imx6dl-cpuB.dts
 create mode 100644 arch/arm/boot/dts/imx6dl-skov-dol63x-revA.dts
 create mode 100644 arch/arm/boot/dts/imx6dl-skov-dol63x-revB.dts
 delete mode 100644 arch/arm/boot/dts/imx6q-cpuA.dts
 delete mode 100644 arch/arm/boot/dts/imx6q-cpuB.dts
 create mode 100644 arch/arm/boot/dts/imx6q-skov-dol63x-revA.dts
 create mode 100644 arch/arm/boot/dts/imx6q-skov-dol63x-revB.dts

diff --git a/arch/arm/boot/dts/imx6dl-cpuA.dts b/arch/arm/boot/dts/imx6dl-cpuA.dts
deleted file mode 100644
index a94e68d6fab9..000000000000
--- a/arch/arm/boot/dts/imx6dl-cpuA.dts
+++ /dev/null
@@ -1,23 +0,0 @@
-/*
- * Copyright 2014 Rose Technology
- * Copyright 2012 Freescale Semiconductor, Inc.
- * Copyright 2011 Linaro Ltd.
- *
- * The code contained herein is licensed under the GNU General Public
- * License. You may obtain a copy of the GNU General Public License
- * Version 2 or later at the following locations:
- *
- * http://www.opensource.org/licenses/gpl-license.html
- * http://www.gnu.org/copyleft/gpl.html
- */
-
-/dts-v1/;
-#include <dt-bindings/interrupt-controller/irq.h>
-#include "imx6dl.dtsi"
-#include "imx6qdl-cpu.dtsi"
-#include "imx6qdl-cpuA.dtsi"
-
-/ {
-	model = "Freescale i.MX6 SKOV Solo Board";
-	compatible = "fsl,imx6dl", "fsl,imx6dl_cpu";
-};
diff --git a/arch/arm/boot/dts/imx6dl-cpuB.dts b/arch/arm/boot/dts/imx6dl-cpuB.dts
deleted file mode 100644
index 38ea3d5ccd1d..000000000000
--- a/arch/arm/boot/dts/imx6dl-cpuB.dts
+++ /dev/null
@@ -1,23 +0,0 @@
-/*
- * Copyright 2014 Rose Technology
- * Copyright 2012 Freescale Semiconductor, Inc.
- * Copyright 2011 Linaro Ltd.
- *
- * The code contained herein is licensed under the GNU General Public
- * License. You may obtain a copy of the GNU General Public License
- * Version 2 or later at the following locations:
- *
- * http://www.opensource.org/licenses/gpl-license.html
- * http://www.gnu.org/copyleft/gpl.html
- */
-
-/dts-v1/;
-#include <dt-bindings/interrupt-controller/irq.h>
-#include "imx6dl.dtsi"
-#include "imx6qdl-cpu.dtsi"
-#include "imx6qdl-cpuB.dtsi"
-
-/ {
-	model = "Freescale i.MX6 SKOV Solo Board";
-	compatible = "fsl,imx6dl", "fsl,imx6dl_cpu";
-};
diff --git a/arch/arm/boot/dts/imx6dl-skov-dol63x-revA.dts b/arch/arm/boot/dts/imx6dl-skov-dol63x-revA.dts
new file mode 100644
index 000000000000..a94e68d6fab9
--- /dev/null
+++ b/arch/arm/boot/dts/imx6dl-skov-dol63x-revA.dts
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2014 Rose Technology
+ * Copyright 2012 Freescale Semiconductor, Inc.
+ * Copyright 2011 Linaro Ltd.
+ *
+ * The code contained herein is licensed under the GNU General Public
+ * License. You may obtain a copy of the GNU General Public License
+ * Version 2 or later at the following locations:
+ *
+ * http://www.opensource.org/licenses/gpl-license.html
+ * http://www.gnu.org/copyleft/gpl.html
+ */
+
+/dts-v1/;
+#include <dt-bindings/interrupt-controller/irq.h>
+#include "imx6dl.dtsi"
+#include "imx6qdl-cpu.dtsi"
+#include "imx6qdl-cpuA.dtsi"
+
+/ {
+	model = "Freescale i.MX6 SKOV Solo Board";
+	compatible = "fsl,imx6dl", "fsl,imx6dl_cpu";
+};
diff --git a/arch/arm/boot/dts/imx6dl-skov-dol63x-revB.dts b/arch/arm/boot/dts/imx6dl-skov-dol63x-revB.dts
new file mode 100644
index 000000000000..38ea3d5ccd1d
--- /dev/null
+++ b/arch/arm/boot/dts/imx6dl-skov-dol63x-revB.dts
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2014 Rose Technology
+ * Copyright 2012 Freescale Semiconductor, Inc.
+ * Copyright 2011 Linaro Ltd.
+ *
+ * The code contained herein is licensed under the GNU General Public
+ * License. You may obtain a copy of the GNU General Public License
+ * Version 2 or later at the following locations:
+ *
+ * http://www.opensource.org/licenses/gpl-license.html
+ * http://www.gnu.org/copyleft/gpl.html
+ */
+
+/dts-v1/;
+#include <dt-bindings/interrupt-controller/irq.h>
+#include "imx6dl.dtsi"
+#include "imx6qdl-cpu.dtsi"
+#include "imx6qdl-cpuB.dtsi"
+
+/ {
+	model = "Freescale i.MX6 SKOV Solo Board";
+	compatible = "fsl,imx6dl", "fsl,imx6dl_cpu";
+};
diff --git a/arch/arm/boot/dts/imx6q-cpuA.dts b/arch/arm/boot/dts/imx6q-cpuA.dts
deleted file mode 100644
index 1d1b94774bd2..000000000000
--- a/arch/arm/boot/dts/imx6q-cpuA.dts
+++ /dev/null
@@ -1,32 +0,0 @@
-/*
- * Copyright 2014 Rose Technology
- * Copyright 2012 Freescale Semiconductor, Inc.
- * Copyright 2011 Linaro Ltd.
- *
- * The code contained herein is licensed under the GNU General Public
- * License. You may obtain a copy of the GNU General Public License
- * Version 2 or later at the following locations:
- *
- * http://www.opensource.org/licenses/gpl-license.html
- * http://www.gnu.org/copyleft/gpl.html
- */
-
-/dts-v1/;
-#include <dt-bindings/interrupt-controller/irq.h>
-#include "imx6q.dtsi"
-#include "imx6qdl-cpu.dtsi"
-#include "imx6qdl-cpuA.dtsi"
-
-/ {
-	model = "Freescale i.MX6 SKOV Quad Board";
-	compatible = "fsl,imx6q", "fsl,imx6_cpu";
-};
-
-&sata {
-	status = "disabled";
-};
-
-&hdmi {
-	status = "okay";
-	ddc-i2c-bus = <&i2c2>;
-};
diff --git a/arch/arm/boot/dts/imx6q-cpuB.dts b/arch/arm/boot/dts/imx6q-cpuB.dts
deleted file mode 100644
index 66c81513b527..000000000000
--- a/arch/arm/boot/dts/imx6q-cpuB.dts
+++ /dev/null
@@ -1,32 +0,0 @@
-/*
- * Copyright 2014 Rose Technology
- * Copyright 2012 Freescale Semiconductor, Inc.
- * Copyright 2011 Linaro Ltd.
- *
- * The code contained herein is licensed under the GNU General Public
- * License. You may obtain a copy of the GNU General Public License
- * Version 2 or later at the following locations:
- *
- * http://www.opensource.org/licenses/gpl-license.html
- * http://www.gnu.org/copyleft/gpl.html
- */
-
-/dts-v1/;
-#include <dt-bindings/interrupt-controller/irq.h>
-#include "imx6q.dtsi"
-#include "imx6qdl-cpu.dtsi"
-#include "imx6qdl-cpuB.dtsi"
-
-/ {
-	model = "Freescale i.MX6 SKOV Quad Board";
-	compatible = "fsl,imx6q", "fsl,imx6_cpu";
-};
-
-&sata {
-	status = "disabled";
-};
-
-&hdmi {
-	status = "okay";
-	ddc-i2c-bus = <&i2c2>;
-};
diff --git a/arch/arm/boot/dts/imx6q-skov-dol63x-revA.dts b/arch/arm/boot/dts/imx6q-skov-dol63x-revA.dts
new file mode 100644
index 000000000000..1d1b94774bd2
--- /dev/null
+++ b/arch/arm/boot/dts/imx6q-skov-dol63x-revA.dts
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2014 Rose Technology
+ * Copyright 2012 Freescale Semiconductor, Inc.
+ * Copyright 2011 Linaro Ltd.
+ *
+ * The code contained herein is licensed under the GNU General Public
+ * License. You may obtain a copy of the GNU General Public License
+ * Version 2 or later at the following locations:
+ *
+ * http://www.opensource.org/licenses/gpl-license.html
+ * http://www.gnu.org/copyleft/gpl.html
+ */
+
+/dts-v1/;
+#include <dt-bindings/interrupt-controller/irq.h>
+#include "imx6q.dtsi"
+#include "imx6qdl-cpu.dtsi"
+#include "imx6qdl-cpuA.dtsi"
+
+/ {
+	model = "Freescale i.MX6 SKOV Quad Board";
+	compatible = "fsl,imx6q", "fsl,imx6_cpu";
+};
+
+&sata {
+	status = "disabled";
+};
+
+&hdmi {
+	status = "okay";
+	ddc-i2c-bus = <&i2c2>;
+};
diff --git a/arch/arm/boot/dts/imx6q-skov-dol63x-revB.dts b/arch/arm/boot/dts/imx6q-skov-dol63x-revB.dts
new file mode 100644
index 000000000000..66c81513b527
--- /dev/null
+++ b/arch/arm/boot/dts/imx6q-skov-dol63x-revB.dts
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2014 Rose Technology
+ * Copyright 2012 Freescale Semiconductor, Inc.
+ * Copyright 2011 Linaro Ltd.
+ *
+ * The code contained herein is licensed under the GNU General Public
+ * License. You may obtain a copy of the GNU General Public License
+ * Version 2 or later at the following locations:
+ *
+ * http://www.opensource.org/licenses/gpl-license.html
+ * http://www.gnu.org/copyleft/gpl.html
+ */
+
+/dts-v1/;
+#include <dt-bindings/interrupt-controller/irq.h>
+#include "imx6q.dtsi"
+#include "imx6qdl-cpu.dtsi"
+#include "imx6qdl-cpuB.dtsi"
+
+/ {
+	model = "Freescale i.MX6 SKOV Quad Board";
+	compatible = "fsl,imx6q", "fsl,imx6_cpu";
+};
+
+&sata {
+	status = "disabled";
+};
+
+&hdmi {
+	status = "okay";
+	ddc-i2c-bus = <&i2c2>;
+};
